services/retrieve_and_rank/v1.js

```diff
@@ -155,8 +155,12 @@ module.exports = function (RED) {
     this.on('input', function(msg) {
       setupRankRetrieveNode(msg, config, this, function (retrieve_and_rank) {
 
+        //Cluster name can be passed into msg.payload, but the cluster name
+        //specified in the config takes priority
+        var clustername;
+        (config.clustername != '') ? clustername = config.clustername : clustername = msg.payload;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '!==' and instead saw '!='.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2661350196/issues/source?bid=3225115&fileBranchId=3329988#l161)